app/controllers/data_vis_controller.rb

```diff
@@ -0,0 +1,4 @@
+class DataVisController < ApplicationController
+    
+    def data_view; end
+end
```

Comment:
Usually before an `end` there is an empty line

```suggestion
    def data_view; end

end
```